src/fullnode.rs

```diff
@@ -900,14 +897,8 @@ mod tests {
         // Wait for convergence
         converge(&leader_node_info, 2);
 
-        info!("Wait for leader -> validator transition");
-        let rotation_signal = leader
-            .rotation_receiver
-            .recv()
-            .expect("signal for leader -> validator transition");
-        debug!("received rotation signal: {:?}", rotation_signal);
-        // Re-send the rotation signal, it'll be received again once the tvu is unpaused
-        leader.rotation_sender.send(rotation_signal).expect("send");
+        // Wait for Tpu bank to progress while the Tvu bank is stuck
+        sleep(Duration::from_millis(1000));
```

Comment:
how is this right? how is 1 second sufficient or too much? Seems kind of random.

Reply:
Without the tpu signal, we no longer have a way to check that the tpu is ahead of the tvu, so the wait is just to ensure the tpu can hit the 5 ticks before the tvu starts again. At the default 10 ticks a second, this should be enough time. What might be better is to set the tick rate for the test to be faster?